lib/ae_page_objects/util/page_polling.rb

```diff
@@ -1,53 +0,0 @@
-module AePageObjects
```

Comment:
Yes, that got messed up a bit. I squashed most of it together now with a better msg.